server/app.py

```diff
@@ -0,0 +1,17 @@
+# app.py
+from flask import Flask, render_template, request, url_for
+import speak
+
+app = Flask(__name__, static_folder="../static/dist", template_folder="../static2/public")
```

Comment:
let's use `app = Flask(__name__, static_folder="../static2/build/static", template_folder="../static2/build")` instead.
And make sure to run `npm run build` from `static2/` before starting Flask.